tools/firely-server-import.rst

```diff
@@ -212,6 +214,17 @@ Your license already permits the usage of FSI if it contains ``http://fire.ly/vo
 Release notes
 -------------
 
+.. _fsi_releasenotes_1.1.0:
+
+Release 1.1.0
+^^^^^^^^^^^^^
+
+* Feature: added support for MongoDb!
+* Feature: added support for performance counters using dotnet-counters. See :ref:`tool_fsi_performance_counters` on how to setup and use dotnet-counters.
+* FSI has been upgraded to .NET 6.
```

Comment:
I would add a note to indicate that you have to install the new SDK, maybe even provide a link to it.